.github/workflows/scripts/check-help-wanted.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
```

Comment:
I'd appreciate a short usage+doc here.

Reply:
I'd be okay if the usage logic below outputs this rather than static docs.  No need to duplicate it and would be helpful to output in dev.